<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KhachHang</class>
 <widget class="QMainWindow" name="KhachHang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
	background-image: url(:/background/BACKGROUND/bg1.jpg);
}</string>
  </property>
  <widget class="QWidget" name="Form">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>43</y>
      <width>861</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #222;
	border-radius: 20px
}

QPushButton
{
	background: red;
	border-radius: 10px;
}

QLineEdit
{
	background: transparent;
	border:none;
	color:#717072;
	border-bottom: 1px solid #717072;
}

QLabel
{
	color: white;
	font: 900 18pt &quot;Segoe UI&quot;;
}

QGroupBox
{
	border: 2px solid rgb(255, 255, 127);
}

#INFO_EDIT
{
		border: 2px solid rgb(255, 255, 127);
		border-radius: 0;
}

#INFO_EDIT_2
{
		border: 2px solid rgb(255, 255, 127);
		border-radius: 0;
}

#INFO_EDIT_3
{
		border: 2px solid rgb(255, 255, 127);
		border-radius: 0;
}

#LEFT_BUTTON
{
	
	border-radius: 20px;
}

#RIGHT_BUTTON
{
	border-radius: 20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="EXIT_BUTTON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 18pt &quot;Segoe UI&quot;;
background-color: rgb(255, 85, 127);</string>
     </property>
     <property name="text">
      <string>GO BACK</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="INFO_EDIT">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>277</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Name:

Price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="DN_LABEL">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>9</y>
       <width>531</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>WELCOME TO             LAPTOP STORE</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="INFO_EDIT_2">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>277</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Name:

Price:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="INFO_EDIT_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>277</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Name:

Price:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BUY_BUTTON">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>390</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;
background-color: rgb(241, 237, 255);
</string>
     </property>
     <property name="text">
      <string>BUY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BUY_BUTTON_2">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>390</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;
background-color: rgb(241, 237, 255);
</string>
     </property>
     <property name="text">
      <string>BUY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BUY_BUTTON_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>390</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;
background-color: rgb(241, 237, 255);
</string>
     </property>
     <property name="text">
      <string>BUY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LEFT_BUTTON">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ing.qrc">
       <normaloff>:/icon/ICON/left-arrow.png</normaloff>:/icon/ICON/left-arrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="RIGHT_BUTTON">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>440</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ing.qrc">
       <normaloff>:/icon/ICON/right-arrow.png</normaloff>:/icon/ICON/right-arrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="PRODUCT_LABEL">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ing.qrc">:/background/BACKGROUND/desktop computer-rafiki.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="PRODUCT_LABEL_2">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>80</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ing.qrc">:/background/BACKGROUND/desktop computer-rafiki.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="PRODUCT_LABEL_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ing.qrc">:/background/BACKGROUND/desktop computer-rafiki.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblMoney">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>460</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiền của bạn:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	
	background-color: rgb(255, 156, 43);
	border-radius: 45px;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ing.qrc">
      <normaloff>:/icon/ICON/laptop.png</normaloff>:/icon/ICON/laptop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lblHello">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ing.qrc"/>
 </resources>
 <connections/>
</ui>
